apps/ashley/templates/services/app/job_01_collectstatic.yml.j2

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "ashley-app-collectstatic-{{ job_stamp }}"
+  namespace: "{{ project_name }}"
+  labels:
+    app: ashley
+    service: app
+    version: "{{ ashley_image_tag }}"
+    job_stamp: "{{ job_stamp }}"
+    # Jobs with the "pre" job type will be executed prior to deployments
+    job_type: "pre"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  template:
+    metadata:
+      name: "ashley-app-collectstatic-{{ job_stamp }}"
+      labels:
+        app: ashley
+        service: ashley
+        version: "{{ ashley_image_tag }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+        job_stamp: "{{ job_stamp }}"
+    spec:
+{% if ashley_image_pull_secret_name is defined and ashley_image_pull_secret_name is not none %}
+      imagePullSecrets:
+        - name: "{{ ashley_image_pull_secret_name }}"
+{% endif %}
+      restartPolicy: Never
+      containers:
+        - name: ashley-collectstatic
+          image: "{{ ashley_image_name }}:{{ ashley_image_tag }}"
+          imagePullPolicy: Always
+          env:
+            - name: DJANGO_SETTINGS_MODULE
+              value: "{{ ashley_django_settings_module }}"
+            - name: DJANGO_CONFIGURATION
+              value: "{{ ashley_django_configuration }}"
+            - name: DB_ENGINE
+              value: "{{ ashley_database_engine }}"
+            - name: DB_NAME
+              value: "{{ ashley_database_name }}"
+            - name: DB_HOST
+              value: "ashley-{{ ashley_database_host }}-{{ deployment_stamp }}"
+            - name: DB_PORT
+              value: "{{ ashley_database_port }}"
```

Comment:
ditto 

And you should also load environment variables from the dot env configMap.